Gems/Atom/Asset/ImageProcessingAtom/Code/Source/ImageLoader/QtImageLoader.cpp

```diff
@@ -57,6 +73,170 @@ namespace ImageProcessingAtom
             return pImage;
         }
 
+        // https://forum.qt.io/topic/74712/qimage-from-tga-with-alpha/11
+        // https://forum.qt.io/topic/101971/qimage-and-tga-support-in-c/5
+        static bool loadTga(const AZStd::string& filename, QImage& image)
+        {
+            if (!image.load(filename.c_str()))
+            {
+                // open the file
+                FILE* pf = nullptr;
+                azfopen(&pf, filename.c_str(), "rb");
+
+                if (!pf)
+                {
+                    return false;
+                }
+
+                // some variables
+                AZStd::vector<uint8_t>* vui8Pixels = nullptr;
+                uint32_t ui32BpP;
+                uint32_t ui32Width;
+                uint32_t ui32Height;
+
+                // read in the header
+                uint8_t ui8x18Header[19] = { 0 };
+
+                if (fread(reinterpret_cast<char*>(&ui8x18Header), sizeof(char), sizeof(ui8x18Header) - 1, pf) != sizeof(ui8x18Header) - 1)
+                {
+                    fclose(pf);
+                    return false;
+                }
+
+                // get variables
+                vui8Pixels = new AZStd::vector<uint8_t>;
+                bool bCompressed;
+                uint32_t ui32IDLength;
+                uint32_t ui32PicType;
+                uint32_t ui32PaletteLength;
+                uint32_t ui32Size;
+
+                // extract all information from header
+                ui32IDLength = ui8x18Header[0];
```

Comment:
I think using a struct for the TGA header would be a better way to ready the data. 

Here is a reference:
http://www.paulbourke.net/dataformats/tga/tgatest.c